rest/route/service.go

```diff
@@ -129,6 +129,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/projects/{project_id}").Version(2).Patch().Wrap(checkUser, addProject, checkProjectAdmin, editProjectSettings).RouteHandler(makePatchProjectByID(sc, env.Settings()))
 	app.AddRoute("/projects/{project_id}").Version(2).Put().Wrap(superUser, createProject).RouteHandler(makePutProjectByID(sc))
 	app.AddRoute("/projects/{project_id}/copy").Version(2).Post().Wrap(checkUser, addProject, checkProjectAdmin, editProjectSettings).RouteHandler(makeCopyProject(sc))
+	app.AddRoute("/projects/{project_id}/copy_redacted").Version(2).Post().Wrap(checkUser, addProject, checkProjectAdmin, editProjectSettings).RouteHandler(makeCopyRedactedVars(sc))
```

Comment:
This route shouldn't require the user to be able to create projects

Reply:
Fixed (although confused why we have both addProject and createProject)

Reply:
Dammit I was confused by the same thing. Addproject just tries to get the project context, so what you had before was right. Sorry!

Reply:
Could you revert the changes you made to rest/route/service in 3c7e47fd318b8bafe0415a3b6a6b2e2547f57cd7? What you had before was the right thing

Reply:
Oh I'm sorry I didn't see this comment--done!